### ISSUE TITLE
NetOptimize::fuse_convolutiondepthwise_batchnorm

### ISSUE THREAD

**Issue by ShuangLiu1992** [Tencent/ncnn#1339]
NetOptimize::fuse_convolutiondepthwise_batchnorm seems to produce incorrect results (regardless of whether group number is the same as input output channels), NetOptimize::fuse_convolution_batchnorm works as expected.

**Comment by ShuangLiu1992** [comment]
more specifically, it is depthwise convolution with bias terms followed by batch norm. I'm retraining a model to see if dep con without bias produce the same result. but looking at ncnnoptimize it shouldn't be.

**Comment by ShuangLiu1992** [comment]
dep convolution without bias seems to not suffer from this, which is very weird... not sure if it's my training setup or something going wrong during conversion. I will retrain to see if I can reproduce this issue and share the model if I have time to retrain a couple of times....

**Comment by nihui** [comment]
change
```
                bias[i] += a[i];
```
to
```
                bias[i] = bias[i] * b[i] + a[i];
```

**Comment by ShuangLiu1992** [comment]
hmmm, should I do that for regular conv as well?

**Comment by nihui** [comment]
apply to all conv/deconv/innerproduct - batchnorm family
Because in most cases, we usually disable the convolution bias when it is followed by batchnorm, so it may not be such serious, but it is definitely a bug ...

**Comment by nihui** [comment]
merged https://github.com/Tencent/ncnn/pull/1340
